frontend/src/pages/CoursesList/CoursesList.tsx

```diff
@@ -12,16 +12,20 @@ import ButtonLoader from 'components/ButtonLoader';
 import { buttonSpinner } from 'animations';
 import { LOADER } from 'constants/loaderTypes';
 import { MobileSearch } from 'components/Layout/MobileSearch';
-import { PAGES } from 'constants/pages';
 import { convertDurationToString } from 'utils/helpers/convertDurationToString';
+import { CustomButton } from 'components/Button/styled';
+import { countProgress } from 'utils/helpers/countCourseProgress';
+import { COURSE_LABELS } from 'constants/statuses';
+import ActionButton from 'components/Button/ActionButton';
+import { COURSE_DISABLE_DAYS } from 'constants/time';
+import getCurrentPageName from 'utils/helpers/getCurentPageName';
+import { chooseListPath } from 'utils/helpers/paths/choosePath';
```

Comment:
Maybe we can add index.ts with exports inside utils/helpers folder to avoid duplication